book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :date
+  attr_reader :rentals
+
+  def initialize(title = 'no title', author = 'no author')
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
```

Comment:
Good job by adding add_rental method to the book class 👍🏼  , but the optimal solution to handle this many to many relation is to create the new rental instance here in this method and in rental initializer push in rentals of book and of person this way when you call Book.add_rental all 'belongs-to' and has-many associations will be created

```ruby
def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```